src/schema/v2/artwork/index.ts

```diff
@@ -133,6 +133,10 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      artworkLocation: {
```

Comment:
Sorry for late comment here. Do I read this correctly - that we've added `artwork_location` here that is different from `location` and is only applicable to myCollection artworks? Maybe worth adding a `description` to this field here. 

Suspect it will be very confusing at some point if somebody looks at artwork and `location` is different than `artworkLocation` and nothing matches `shippingOrigin`.

Apologies if it has been discussed and resolved already.

Reply:
Good point. I will add a description. Thanks